.github/workflows/milestone.yml

```diff
@@ -0,0 +1,61 @@
+name: Check PR milestone
+
+on:
+  pull_request_target:
+    types: [milestoned, demilestoned, opened, reopened]
+    branches:
+      - main
```

Comment:
do we need to specify a branch, when we trigger on PR?

Reply:
I think this specifies the PR target branch in this context.